### DISCUSSION TITLE
Could the download .appimage and .deb files have an issue?

### DISCUSSION THREAD

**Discussion by tomhusband** [artisan-roaster-scope/artisan#1517] [Q&A] [answered]
I don't want to call it a bug as it might be on my end but I've been trying to install Artisan on a new (to me) Linux Mint laptop and it seems to install but it won't open. Both the appimage and deb files do nothing when I double click. Nothing happens when I try to open them in terminal either. No error message. 

I asked the question in the Linux Mint forum thinking it was something on my end but another forum member said it wouldn't open for them either.

**Comment by roasterdave** [comment, marked as answer]
Please try the [continuous build](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous) and let us know.

**Reply by tomhusband** [reply]
Yes,that seems to work.

Many thanks!
